### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 1

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#177] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152252098-9b9d628e-0ecc-46ec-8cf2-5bc7c3ce37e2.png)

**Comment by AxnNxs** [comment]
Per il Principio Moltiplicativo, tutte le possibili combinazioni del tipo "pane - farcitura" è data da:

5 x 8 = 40
